Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```diff
@@ -219,6 +219,17 @@
         acts: [ "Destruction" ]
       - !type:DumpCanisterBehavior
 
+- type: entity
```

Comment:
these should all be locked so shitters cant do homemade frezonloose

Reply:
> these should all be locked so shitters cant do homemade frezonloose

you can still break canister to troll

Reply:
accessreader still exists for a reason